modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,15 @@
+== Inline formatting.
+
+List bullets and list numbers can be spot- or inline-formatted.
+
+If an end-user selects the contents of a list item and applies inline formatting, such as a change of color or change of font-size, this spot formatting is also applied to the list item’s associated bullet or number.
+
+Only the bullets or list numbers associated with the selection are inline-formatted.
+
+NOTE: PowerPaste does not, currently, support inline-formatted bullets or list numbers. Inline-formatting, as applied to the bullets or list numbers, is lost when such material is copied and pasted.
+
+If the selection is one list item (ie, if the selection runs from an particular `<li>` tag to its associated `</li>` tag), only the bullet or number associated with the single selected list item takes on the inline-formatting.
+
+If the selection is the entire list, (ie, if the selection runs from the list’s opening `<ol>` or `<ul>` tag to the closing `</ol>` or `</ul>` tag, or if the selection runs from the list’s first `<li>` tag to the last `</li>` tag), every bullet or number takes on the inline-formatting.
```

Comment:
As above, this reads like just the bullets / numbers change. Although, interestingly, less so.

Reply:
To quote my commit message:

> Edited multiple paragraphs to make explicit the relationship between spot-formatting a selection and the bullets or list numbers that will, consequently, take on the equivalent spot-formatting.